web/src/gamestats/population-current.html

```diff
@@ -42,6 +55,8 @@
         'drawCurrentPopulationChart(chartLoaded, game)',
       ],
 
+      lastResize: null,
```

Comment:
this makes me nervous because ive never seen variables made like this... can you do me a favor and move it to a 'this.lastResize = null' in attached, or perhaps make it a property?